src/main/resources/application-docker.properties

```diff
@@ -1,24 +1,45 @@
 ### App setup
 spring.banner.location=classpath:true-connector-banner.txt
 
+logging.config=classpath:logback-SENDER.xml
```

Comment:
Not sure that this property file is used, but good thing to have it aligned
We might create ticket to check if we can delete it